packages/stark-ui/src/modules/action-bar/components/_action-bar-theme.scss

```diff
@@ -1,38 +1,23 @@
 .stark-action-bar {
-  .mat-icon-button {
-    @include stark-button-color(mat-color(map-get($base-theme, primary-palette)));
+  .action-label {
+    color: mat-color(map-get($base-theme, primary-palette));
+    &.disabled {
+      color: map-get($mat-light-theme-foreground, disabled-text);
+    }
   }
-  &.stark-action-bar-full {
+}
+
+@media #{$mobile-only-query} {
```

Comment:
In the previous code the media query was "tablet-query" and now is "mobile-only-query"... Is this an intended change? Why? What about tablet?

Reply:
Yes it is intended, now with the button theme, we can specify which color the button should have with the color attribute. It is only an small device that we need to use custom colors, so it make more sense to have SCSS exception on small device

Reply:
Oh I see, Ok.